src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -67,16 +71,20 @@ const AudioInputProvider: React.FC = ({ children }) => {
           console.log(
             'Previously selected audio input lost. Selecting a default device.'
           );
-          meetingManager.selectAudioInputDevice(newAudioInputs[0].deviceId);
+          await meetingManager.selectAudioInputDevice(newAudioInputs[0].deviceId);
```

Comment:
Why does this not need similar reselection handling?

Reply:
Good catch, will add the same logic here